REFERENCE.md

```diff
@@ -274,6 +274,24 @@ For further discussion of the overlap assumption, please see Imbens and Rubin (2
 
 While the algorithm in `regression_forest` is very similar to that of classic random forests, it has several notable differences, including 'honesty', group tree training for variance estimates, and restrictions during splitting to avoid imbalanced child nodes. These features can cause the predictions of the algorithm to be different, and also lead to a slower training procedure than other packages. We welcome GitHub issues that shows cases where GRF does notably worse than other packages (either in statistical or computational performance), as this will help us choose better defaults for the algorithm, or potentially point to a bug.
 
+
+### Forests predict different values depending on the platform even though the seed is the same
+
+When it comes to cross-platform predictions, the output of `grf` forest will depend on a few factors beyond the forest seed.
```

Comment:
I would be nice to add a couple opening sentences to give context: "Overall, GRF is designed to produce the same estimates across platforms when using a consistent value for the random seed through the training option `seed`. However, there are still some cases where GRF can produce different estimates across platforms."

Reply:
Sure, The reason I was hesitant to include such a statement is because the user _must_ set `num.threads` to the same number as well. Otherwise, we get different results even if the other configurations are the same.

Reply:
To me it's still accurate + helpful to say 'Overall, GRF is designed to produce the same estimates...'. That way the user knows it should be possible to get consistent estimates after making a few adjustments (in contrast with some software where there is not even an attempt at consistency).

Reply:
Ah, I see where you're coming from then. That's fair. The sentence is in now!